ports/cxd56/supervisor/port.c

```diff
@@ -100,11 +108,11 @@ uint32_t *port_stack_get_top(void) {
 }
 
 uint32_t *port_heap_get_bottom(void) {
-    return port_stack_get_limit();
+    return heap;
```

Comment:
Unfortunately I don't think this will do what you want necessarily because this heap region also contains supervisor allocations in addition to the MicroPython heap. Are you sure this is needed once the heap block size is set?

Reply:
It is better to do it like this because now I can allocate all available memory.